examples/tensorflow_mnist.py

```diff
@@ -14,10 +14,20 @@
 # ==============================================================================
 #!/usr/bin/env python
 
+import os
+
 import tensorflow as tf
 import horovod.tensorflow as hvd
-layers = tf.contrib.layers
-learn = tf.contrib.learn
+import numpy as np
+
+from distutils.version import LooseVersion
+
+if LooseVersion(tf.__version__) >= LooseVersion("1.4.0"):
```

Comment:
I actually think it's OK to assume reasonably fresh TF version in examples - so, just doing tf.keras should be OK (and cleaner).